### DISCUSSION TITLE
enable-proposed-api GitHub.copilot

### DISCUSSION THREAD

**Discussion by 0x11c11e** [community/community#13948] [Copilot Conversations]
editor/inlineCompletions/actions is a proposed menu identifier. It requires 'package.json#enabledApiProposals: ["inlineCompletions"]' and is only available when running out of dev or with the following command line switch: --enable-proposed-api GitHub.copilot.

- debian

**Comment by davecheney** [comment]
Im sorry but copilot only works with VScode. It does not currently support VSCodium or OSSCode.

**Reply by GMishx** [reply]
What does this error actually mean? Can we pass the flag --enable-proposed-api somehow to enable it?

**Reply by JackDunnCodes** [reply]
Why does it only work with VSCode? VSCodium is using essentially the same code, as far as I'm aware

**Reply by alko89** [reply]
I'm also confused about this, I was using Copilot for months now without issues and now it suddenly break.

**Comment by MunirG05** [comment]
I did `code . --enable-proposed-api` and it worked

**Reply by alko89** [reply]
I haven't tried, but it worked setting `"enable-proposed-api":["github.copilot"]` in runtime arguments:

https://github.com/github/feedback/discussions/6629#discussioncomment-1524627

**Reply by acheong08** [reply]
This solution appeared to work but has since been patched. It no longer works.

**Reply by acheong08** [reply]
Which versions are you on? Vscodium or OSS vscode?

**Reply by acheong08** [reply]
`codium --enable-proposed-api GitHub.copilot`
results in
```
editor/inlineCompletions/actions is a proposed menu identifier. It requires 'package.json#enabledApiProposals: ["inlineCompletionsAdditions"]' and is only available when running out of dev or with the following command line switch: --enable-proposed-api GitHub.copilot
```

**Reply by MunirG05** [reply]
Sorry for the slow reply, I'm on OSS VSCode

**Reply by acheong08** [reply]
> "enable-proposed-api":["github.copilot"]

Unknown Configuration Setting

**Reply by hristo-mavrodiev** [reply]
`code . --enable-proposed-api GitHub.copilot --extensions-dir=<path_to_extensions_dir>` <
where path_to_extensions_dir is where Github copilot extension is located - `~/.vscode-oss/extensions/`

**Comment by Leone25** [comment]
The solution for this is to go to `~/.vscode-oss/argv.json`, and add `"enable-proposed-api": ["GitHub.copilot"]`, then restart vs code

**Comment by Lan-ce-lot** [comment]
> I did `code . --enable-proposed-api` and it worked

For solving the problem, thank you very much.
